packages/button/src/ButtonStyles.tsx

```diff
@@ -178,7 +178,7 @@ export const ButtonStyle = styled.button<InjectedProps>`
   position: relative;
   background: transparent;
   font-family: inherit;
-  font-weight: 800;
+  font-weight: 700;
```

Comment:
Discussed with Stanislav, he said font-weight for buttons should be 700, not 800. It's ok that there would not be backward compatibility for this. [Proofs](https://www.figma.com/file/0j4EJ3Ra7OPBeEKLXFQiGM/LIDO-Design-System?node-id=10497%3A2582)